src/KernelStrategy.sol

```diff
@@ -131,6 +131,17 @@ contract KernelStrategy is Vault {
         return balanceOf(owner);
     }
 
+    /**
+     * @notice Previews the amount of assets that would be received for a given amount of shares.
+     * @param shares The amount of shares to redeem.
+     * @return assets The equivalent amount of assets.
+     */
+    function previewRedeem(uint256 shares) public view override returns (uint256 assets) {
```

Comment:
why do this override?

Reply:
This is to ensure `previewRedeem(uint256 shares)` behaves the same way as 
`previewRedeemAsset(address asset_, uint256 shares)` with the asset being `asset()`

Reply:
well it shouldn't, these 2 have 2 different semantics right